source/MaterialXGenOsl/OslSyntax.cpp

```diff
@@ -246,6 +246,68 @@ class OSLMatrix3TypeSyntax : public AggregateTypeSyntax
     }
 };
 
+class FileFormatSyntax : public AggregateTypeSyntax
+{
+  public:
+    FileFormatSyntax(const string& name, const string& defaultValue, const string& uniformDefaultValue,
+                     const string& typeAlias = EMPTY_STRING, const string& typeDefinition = EMPTY_STRING,
+                     const StringVec& members = EMPTY_MEMBERS) :
+        AggregateTypeSyntax(name, defaultValue, uniformDefaultValue, typeAlias, typeDefinition, members)
+    {}
+    
+    string getValue(const ShaderPort* port, bool uniform) const override
+    {
+        if (!port)
+        {
+            return EMPTY_STRING;
+        }
+
+        StringVec values;
+        values.push_back(port->getValue() ? port->getValue()->getValueString() : EMPTY_STRING);
+        values.push_back(port->getColorspace());
+        return getValue(values, uniform);
+    }
+
+    string getValue(const Value& value, bool uniform) const override
+    {
+        StringVec values;
+        values.push_back(value.getValueString());
+        values.push_back(EMPTY_STRING); // TODO: How to get color space initialized
+
+        return getValue(values, uniform);
```

Comment:
Here it's even easier to construct and return the value string directly, without using the utility method.